requirements.txt

```diff
@@ -16,4 +16,4 @@ six==1.16.0
 sqlparse==0.2.4
 python-ldap==3.1.0
 django-auth-ldap==2.2.0
-
+selenium
```

Comment:
Se deberia especificar la versión de selenium, se pueden ver todas las versiones disponibles en https://pypi.org/project/selenium/#history

Reply:
Added version 4.1.0 to selenium